packages/scripts/webapp/preset/config/webpack.config.js

```diff
@@ -119,6 +119,24 @@ module.exports = ({ getUserConfig, mode }) => {
 						mimetype: 'application/font-woff',
 					},
 				},
+				{
+					test: /\.svg(\?v=\d+\.\d+\.\d+)?$/,
```

Comment:
Could you explain the query string here? Where does it come from?

Reply:
Oh in fact I copied/pasted that from TMC :D 
I'll fix it. Thanks

Reply:
Oh in fact I'm guilty of copy/pasting from TMC :D 
I'll fix it. Thanks